quickstart/targets.go

```diff
@@ -0,0 +1,79 @@
+package quickstart
+
+import (
+	"fmt"
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/huh"
+	"github.com/pkg/errors"
+	"github.com/speakeasy-api/openapi-generation/v2/pkg/generate"
+	"github.com/speakeasy-api/sdk-gen-config/workflow"
+	"github.com/speakeasy-api/speakeasy/charm"
+)
+
+func targetBaseForm(inputWorkflow *workflow.Workflow) (*State, error) {
+	var targetName, targetType, sourceName, outputLocation string
+	if _, err := tea.NewProgram(charm.NewForm(huh.NewForm(
+		huh.NewGroup(
```

Comment:
The styling of this section is a bit confusing. It's weird because the different "steps" aren't really separated visually. It looks like I should be able to start selecting a language right away. Can we gray the inactive steps out? Or maybe "collapse" them somehow

![Screen Shot 2024-01-09 at 1 41 06 PM](https://github.com/speakeasy-api/speakeasy/assets/90289500/bff044b4-7d00-41f5-99e9-9144451fcc3f)

Reply:
Hold off on styling for a sec. We could make these different forms if we want but I kind of liked having them on one page